internetarchive/iaux-icons
# new icon: icon-close-circle

+ added to ia-icons

To Test:
- pull down repo -> go to directory: packages/ia-icons
- in `packages/ia-icons` directory, run command in-terminal: `npm install && npm start`
- local server pops up with icon list (random colors generated)
- you should see `closeIcon` in the list (like image)
![image](https://user-images.githubusercontent.com/7840857/118685911-5b62d000-b7b8-11eb-9506-2a2ae96897a0.png)
